gro/__init__.py

```diff
@@ -0,0 +1,4 @@
+from gro import lib
+from gro.base_client import Client
+from gro.batch_client import BatchClient
```

Comment:
Is there a way we can do this without adding BatchClient to __init__? Reason being that BatchClient depends on third party library (Tornado) which isn't required for normal client, but in this case gets imported nonetheless. 

I suppose the alternative is to rename the files themselves to allow a 'native' import (not sure what to call it, an import without having to use the dot)

Reply:
I can put each of these in try/except blocks so if you don't have the dependency it doesn't fail - it just doesn't import that client. Like if you don't have pandas, GroClient can fail, and if you don't have tornado BatchClient can fail. In that case you could still use Client or lib if you wanted.

Is worth noting that tornado is in the requirements.txt https://github.com/gro-intelligence/api-client/blob/development/requirements.txt#L9 so you should have it already.

Aside from that, I'm not sure how to avoid having that extra redundant `batch_client` layer in `from gro.batch_client import BatchClient`

Reply:
^ I added the exception catching mentioned above, and now, having specifically uninstalled tornado, this is my output:

```shell
$ python quick_start.py

Not able to import BatchClient: No module named 'tornado'
{'start_date': '2019-01-01T00:00:00.000Z', 'end_date': '2019-12-31T00:00:00.000Z', 'value': 3.079994034538774, 'input_unit_id': 61, 'input_unit_scale': 1, 'metric_id': 170037, 'item_id': 270, 'region_id': 1215, 'frequency_id': 9, 'unit_id': 61}
```

Reply:
> I can put each of these in try/except blocks so if you don't have the dependency it doesn't fail - it just doesn't import that client. Like if you don't have pandas, GroClient can be excluded, and if you don't have tornado BatchClient can be excluded. In that case you could still use Client or lib if you wanted.
OK this works. My original thought was to avoid importing all of the API classes when you import one part of the API. But overhead shouldn't be too bad with these three things. I think this is good for now.

Reply:
lgtm